org-bookmark-heading.el

```diff
@@ -130,6 +133,16 @@ heading.  Set org-id for heading if necessary."
 			    (handler . ,handler)
 			    (front-context-string . ,front-context-string)))))
 
+(defun org-bookmark-heading-default-name (filename heading)
```

Comment:
Nitpick: Let's call it `org-bookmark-heading-default-name-fn`.  I'm not sure, but I think that's more consistent with other instances I've seen of the pattern of having a custom function and a default choice.